code/modules/mining/equipment/survival_pod.dm

```diff
@@ -142,6 +142,8 @@
 	. = ..()
 	if(!state_open)
 		. += "sleeper_cover"
```

Comment:
This is an overlay, not an icon state.
What's changing their icon state?

Reply:
i think uh
```
	if(state_open)
		icon_state = "[initial(icon_state)]-open"
```
from the original sleeper was messing it up? i just did what i can to fix it

Reply:
Survival pod's ``update_icon_state`` should be added instead then, call parent, and set it to ``"[initial(icon_state)]_cover"``

Reply:
Yeah you're gonna have to give me exact code since everything I try is keeping it invisible or changing the full icon state to sleeper_cover instead of adding it over the top.

Reply:
I made a fix of it to https://github.com/yogstation13/Yogstation/pull/20041